src/lib/supported-node-js-version-check.js

```diff
@@ -0,0 +1,46 @@
+// Ref. https://nodejs.org/en/about/releases
+const nodeVersionToEndOfLifeDateMap = {
+	'12': new Date('2022-04-30'),
+	'14': new Date('2023-04-30'),
+	'16': new Date('2023-09-11'),
+	'18': new Date('2025-04-30'),
+	'20': new Date('2026-04-30')
+};
+
+/**
+ * @param {string} appName
+ * @returns {import("../../typings/metrics").Healthcheck}
+ */
+module.exports = (appName) => {
+	const lastUpdated = new Date();
+
+	const currentDate = new Date();
+
+	const region = process.env.REGION === 'US' ? 'us' : 'eu';
+
+	const nodeVersion = process.versions.node;
+
+	const nodeMajorVersionRegex = new RegExp(/^([0-9]*)\./);
+
+	const nodeMajorVersion = nodeVersion.match(nodeMajorVersionRegex)[1];
+
+	const hasSupportedVersion = nodeVersionToEndOfLifeDateMap[nodeMajorVersion] > currentDate;
+
+	return {
+		getStatus: () => {
+			return {
+				id: `supported-node-js-version-${region}`,
+				name: `Long-term supported (LTS) Node.js version used for ${appName} (${region.toUpperCase()})`,
+				ok: hasSupportedVersion,
+				checkOutput: hasSupportedVersion
+					? `${appName} uses v${nodeVersion} of Node.js, which is a long-term supported (LTS) version`
+					: `${appName} uses v${nodeVersion} of Node.js, which is not a long-term supported (LTS) version`,
+				lastUpdated,
+				severity: 3,
+				businessImpact: 'Systems may be subject to security and/or functional bugs that will not be fixed and which could manifest in various ways.',
+				technicalSummary:`${appName} is not using a long-term supported version of Node.js and so will not receive critical bug fixes.`,
```

Comment:
**suggestion:** I think technical summary kind of duplicates the business impact here, [technical summary](https://docs.google.com/document/d/1ftlkDj1SUXvKvKJGvoMoF1GnSUInCNPnNGomqTpJaFk/edit#heading=h.ub010a1bgfy8) should explain what the check is testing I think, so maybe something like:

```suggestion
				technicalSummary: 'Inspects the version of Node.js running the application and verifies that the end-of-life date for it has not passed',
```

Reply:
👍 Updated in commit [Add supported Node.js version healthcheck](https://github.com/Financial-Times/n-express/pull/718/commits/47faf2ec52439dbe003d678639c59217249aa266).